src/main/java/de/rabea/game/Game.java

```diff
@@ -46,7 +27,7 @@ public void play(int boardDimension) {
         }
         finishGame(currentPlayer.mark(), board);
         if (userInterface.playAgain()) {
-            setUpNewGame();
+            gameSetup.setUpNewGame();
```

Comment:
@felipesere OK, ignore my previous comment... I spoke to Jim about this bit of the code. I have a story to get rid of the circular dependency between Game and GameSetup, so I think that will eliminate this code smell.